README.md

```diff
@@ -109,7 +109,8 @@ Currently we don't have a playbook which installs KubeVirt on a Kubernetes clust
 ### OpenShift cluster
 
 Install KubeVirt on your OpenShift cluster.
-[KubeVirt parameters doc](https://github.com/rthallisey/kubevirt-ansible/blob/master/roles/kubevirt/README.md)
+
+See [KubeVirt parameters documentation](./roles/kubevirt/README.md) for more details on parameters.
```

Comment:
@rthallisey Please move this line after the command which uses these parameters. Command's options/variables explanation should go after a command, not before.